packages/sdk/source/ledger.contract.ts

```diff
@@ -21,7 +21,7 @@ export interface LedgerService {
 	signMessage(path: string, payload: string): Promise<string>;
 
 	scan(options?: {
-		useLegacy: boolean;
+		useLegacy?: boolean;
```

Comment:
let's discuss this prior to continuing this PR. Mainsail will be "new" in the sense that it requires a hard fork, but existing data from ARK will still need to be retrievable as old addresses (including ledger) need to be compatible